lib/makeBuild.js

```diff
@@ -15,6 +15,7 @@ function makeBuild(shortBuild) {
     parseWebpackConfig(shortBuild.webpack, shortBuild.name))
 }
 
+// TODO: is this function really necessary?
```

Comment:
Good catch, when I wrote this initially it seemed like there would be more things different between the 'shortBuild' (shorthand webpack config object) and what `build` and `executeBuild` would want to use. Probably safe to remove